### ISSUE TITLE
Descriptive error when host machine doesn't have accessibility turned on

### ISSUE THREAD

**Issue by moredip** [moredip/Frank#41]
see http://groups.google.com/group/frank-discuss/browse_thread/thread/873475cea22280ab
